src/usage/compatibility.md

```diff
@@ -0,0 +1,81 @@
+# Pgvector compatibility
+
+Pgvecto.rs is natively compatible with pgvector at:
+- Create table
+- Search vectors
+
+Pgvecto.rs can be configured to be compatible with pgvector at: 
+- Create vector indexes
+
+This feature is called `compatibility mode`. 
+
+It can be enabled with `SET vectors.pgvector_compatibility=on;`.
```

Comment:
I prefer enable_* in fact because it keeps same style with PostgreSQL GUCs.

Reply:
It would be too longer like `enable_pgvector_compatibility`, actually I referred to https://github.com/lanterndata/lantern#a-note-on-operators-and-operator-classes